docs/arch/monitor.md

```diff
@@ -0,0 +1,47 @@
+# Monitor
+
+## Contributors
+
+- Sarah THEOULLE
+- Pauline CONTAT
+- Thomas BROINE
+- Baptiste BRONSIN
+
+## Functionnalities
+
+- Listening events on a Git remote repository
+- Recognition of the event and adapt the action according to the type of the event
+- Calling the controller through an external API
+
+## What
+
+A monitor must listen to some events from a git remote repository and do some actions onto it.
+We need to recognise the type of event, a `PR` (Pull Request) at first.
+Depending on the type of the event, a yaml data will be generated and sent to the controller.
+
+## Why
+
+The goal is to trigger the controller to launch a CI accordingly to the detected event from the distant repository.
+The `yaml` format has been chosen because the controller requires this type of file.
+
+## How
+
+![schema of the monitor gobal architecture](image-1.png)
```

Comment:
Maybe use mermaid for charts but not mandatory at all (would look better though)

Reply:
I deleted this schema.